File: src/main/java/architecture/goldenboughs_lib/datagen/LibDatagenBlockState.java
package architecture.goldenboughs_lib.datagen;

import architecture.goldenboughs_lib.core.Lib;
import net.minecraft.data.PackOutput;
import net.neoforged.neoforge.client.model.generators.BlockStateProvider;
import net.neoforged.neoforge.common.data.ExistingFileHelper;

public final class LibDatagenBlockState extends BlockStateProvider {
	public LibDatagenBlockState(PackOutput output, ExistingFileHelper exFileHelper) {
		super(output, Lib.ID, exFileHelper);
	}

	@Override
	protected void registerStatesAndModels() {

	}
}

File: src/main/kotlin/architecture/goldenboughs_lib/datagen/LibDatagenBlockState.kt
package architecture.goldenboughs_lib.datagen

import architecture.goldenboughs_lib.util.LibUtil
import net.minecraft.data.PackOutput
import net.neoforged.neoforge.client.model.generators.BlockStateProvider
import net.neoforged.neoforge.common.data.ExistingFileHelper

class LibDatagenBlockState(output: PackOutput, exFileHelper: ExistingFileHelper) :
	BlockStateProvider(output, LibUtil.ID, exFileHelper) {
	override fun registerStatesAndModels() {
	}
}
